src/schemas/21-686C/schema.js

```diff
@@ -1,18 +1,482 @@
+  { value: "MD", label: "Maryland"},
+  { value: "ME", label: "Maine"},
+  { value: "MH", label: "Marshall Islands"},
+  { value: "MI", label: "Michigan"},
+  { value: "MN", label: "Minnesota"},
+  { value: "MO", label: "Missouri"},
+  { value: "MP", label: "Northern Mariana Islands"},
+  { value: "MS", label: "Mississippi"},
+  { value: "MT", label: "Montana"},
+  { value: "NC", label: "North Carolina"},
+  { value: "ND", label: "North Dakota"},
+  { value: "NE", label: "Nebraska"},
+  { value: "NH", label: "New Hampshire"},
+  { value: "NJ", label: "New Jersey"},
+  { value: "NM", label: "New Mexico"},
+  { value: "NV", label: "Nevada"},
+  { value: "NY", label: "New York"},
+  { value: "OH", label: "Ohio"},
+  { value: "OK", label: "Oklahoma"},
+  { value: "OR", label: "Oregon"},
+  { value: "PA", label: "Pennsylvania"},
+  { value: "PI", label: "Philippine Islands"},
+  { value: "PR", label: "Puerto Rico"},
+  { value: "PW", label: "Palau"},
+  { value: "RI", label: "Rhode Island"},
+  { value: "SC", label: "South Carolina"},
+  { value: "SD", label: "South Dakota"},
+  { value: "TN", label: "Tennessee"},
+  { value: "TX", label: "Texas"},
+  { value: "UM", label: "United States Minor Outlying Islands"},
+  { value: "UT", label: "Utah"},
+  { value: "VA", label: "Virginia"},
+  { value: "VI", label: "US Virgin Islands"},
+  { value: "VT", label: "Vermont"},
+  { value: "WA", label: "Washington"},
+  { value: "WI", label: "Wisconsin"},
+  { value: "WV", label: "West Virginia"},
+  { value: "WY", label: "Wyoming"}
+]
+
+const textRegex = '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$';
+const textAndNumbersRegex = '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*]+$';
+
+let definitions = _.cloneDeep(originalDefinitions);
+definitions =  _.pick(definitions, ['fullName']);
+
+definitions.fullName.properties.first.pattern = textRegex
+definitions.fullName.properties.last.pattern = textRegex
+definitions.fullName.properties.middle.pattern = textRegex
+
+const commonAddressFields = {
+  required: ['addressType', 'street', 'city', 'country'],
+  properties: {
+    addressType: {
+      type: 'string',
+      'enum': [
+        'DOMESTIC',
+        'MILITARY',
+        'INTERNATIONAL'
+      ],
+      enumNames: [
+        'Domestic',
+        'Military',
+        'International'
+      ]
+    },
+    street: {
+      type: 'string',
+      minLength: 1,
+      maxLength: 20,
+      pattern: textAndNumbersRegex
+    },
+    street2: {
+      type: 'string',
+      maxLength: 20,
+      pattern: textAndNumbersRegex
+    },
+    street3: {
+      type: 'string',
+      maxLength: 20,
+      pattern: textAndNumbersRegex
+    },
+    city: {
+      type: 'string',
+      maxLength: 30,
+      pattern: textRegex
+    },
+    country: {
+      type: 'string',
+      maxLength: 50,
+      'enum': [countryUSA, countryMilitary, countryNotInList].concat(countriesForeign)
+    }
+  }
+}
+
+const postalCode = {
+  type: 'string',
+  maxLength: 10,
+  pattern: '^[0-9]{5,10}$'
```

Comment:
I might be wrong but if the postalCode needs to be 5 or 10 characters, then a `-` is also a valid character since 10 digit zips are in the format: 12345-6789. Maybe this pattern should really be two patterns: one for 5 digit zips and one for zip+4.

Reply:
I had similar thoughts about a dash, but forgot to follow up.
In their documentation it is defined as 
```
The maximum amount of characters for this field is 10. Please only enter numeric values.
```